src/dom/DomEvent.DoubleTap.js

```diff
@@ -17,9 +17,7 @@ L.extend(L.DomEvent, {
 			var count;
 
 			if (L.Browser.pointer) {
-				if ((!L.Browser.edge) || e.pointerType === 'mouse') {

```

Comment:
This code won't run on anything but Edge anyway (Chrome+pointer listens only for native `dblclick`s), so the extra check is not needed.